tox.ini

```diff
@@ -24,17 +24,17 @@ pip_pre = true
 # side effects and make sure all test suites pass with all packages
 deps =
     astropy[all,test]
-    asdf_astropy,all: git+https://github.com/astropy/asdf-astropy.git#egg=asdf_astropy[test]
-    astropy_healpix,all: git+https://github.com/astropy/astropy-healpix.git#egg=astropy_healpix[test]
-    astroquery,all: git+https://github.com/astropy/astroquery.git#egg=astroquery[test,all]
+    asdf_astropy,all: asdf_astropy[test] @ git+https://github.com/astropy/asdf-astropy.git
+    astropy_healpix,all: astropy_healpix[test] @ git+https://github.com/astropy/astropy-healpix.git
+    astroquery,all: astroquery[test,all] @ git+https://github.com/astropy/astroquery.git
     ccdproc,all: psutil
```

Comment:
Oops did I use the wrong syntax to pull in dependencies? My bad!

Reply:
I think this original syntax did work at some point but I think it's not the latest and maybe doesn't work with new features like what sunpy uses. In any case, this should now be the preferred syntax.